auctioneer/js/idl/auctioneer.json

```diff
@@ -731,6 +743,18 @@
           {
             "name": "reservePrice",
             "type": "u64"
+          },
```

Comment:
I figured their default values of 0 took care of it and the SDK would just send down the default unless otherwise specified, but you're right, options probably make more sense in the Rust code.

Reply:
OK, done.